src/System.Windows.Extensions/tests/X509Certificate2UIManualTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{   
+    // Run these tests after setting the MANUAL_TESTS environment variable.
+    public class X509Certificate2UIManualTests
+    {
+        public static bool ManualTestsEnabled => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MANUAL_TESTS"));
+
+        [ConditionalTheory(nameof(ManualTestsEnabled))]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(2)]
+        public static void SelectCertificateSingleSelectionTest(int count)
+        {
+            using (ECDsa ecdsa = ECDsa.Create(EccTestData.Secp256r1Data.KeyParameters))
+            {
+                CertificateRequest request = new CertificateRequest("CN=Test", ecdsa, HashAlgorithmName.SHA256);
+                DateTimeOffset now = DateTimeOffset.UtcNow;
+
+                using (X509Certificate2 cert = request.Create(request.SubjectName, X509SignatureGenerator.CreateForECDsa(ecdsa), now, now.AddMinutes(10), new byte[1]))
+                {
+                    X509Certificate2Collection collection = new X509Certificate2Collection() { cert, cert };
```

Comment:
Should there be comments in the test describing what I'm supposed to do manually?

Reply:
Actually I added that as the title of the pop-up box
```
collection, $"Choose {count} Certificate", string.Empty,
```